package.json

```diff
@@ -1,75 +1,58 @@
 {
-  "name": "integreat-cms",
```

Comment:
I would suggest that we keep the "name" field as it seems to be generated every time we run the installation procedure.